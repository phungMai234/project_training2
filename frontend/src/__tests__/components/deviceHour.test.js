import React from 'react';
import {Provider} from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import DeviceHour from "../../components/deviceHour/deviceHour";

const mockStore = configureStore([]);

describe('Device hour Component', () => {
  let store;
  let component;
  beforeEach(() => {
    store = mockStore({
      query: {
        data: {
          startDate: "2019/04/29",
          endDate: "2019/04/29"
        }
      },
      hour: {
        data: [{
          "name": "Monday",
          "data": [{"x": "0:00", "y": 4}, {"x": "1:00", "y": 34}, {"x": "2:00", "y": 1}, {
            "x": "3:00",
            "y": 0
          }, {"x": "4:00", "y": 13}, {"x": "5:00", "y": 15}, {"x": "6:00", "y": 13}, {
            "x": "7:00",
            "y": 13
          }, {"x": "8:00", "y": 14}, {"x": "9:00", "y": 21}, {"x": "10:00", "y": 29}, {
            "x": "11:00",
            "y": 4
          }, {"x": "12:00", "y": 16}, {"x": "13:00", "y": 5}, {"x": "14:00", "y": 11}, {
            "x": "15:00",
            "y": 9
          }, {"x": "16:00", "y": 20}, {"x": "17:00", "y": 16}, {"x": "18:00", "y": 17}, {
            "x": "19:00",
            "y": 42
          }, {"x": "20:00", "y": 1}, {"x": "21:00", "y": 14}, {"x": "22:00", "y": 10}, {"x": "23:00", "y": 4}]
        }, {
          "name": "Tuesday",
          "data": [{"x": "0:00", "y": 23}, {"x": "1:00", "y": 12}, {"x": "2:00", "y": 26}, {
            "x": "3:00",
            "y": 20
          }, {"x": "4:00", "y": 35}, {"x": "5:00", "y": 5}, {"x": "6:00", "y": 6}, {"x": "7:00", "y": 15}, {
            "x": "8:00",
            "y": 17
          }, {"x": "9:00", "y": 14}, {"x": "10:00", "y": 17}, {"x": "11:00", "y": 3}, {
            "x": "12:00",
            "y": 28
          }, {"x": "13:00", "y": 44}, {"x": "14:00", "y": 29}, {"x": "15:00", "y": 6}, {
            "x": "16:00",
            "y": 34
          }, {"x": "17:00", "y": 17}, {"x": "18:00", "y": 9}, {"x": "19:00", "y": 14}, {
            "x": "20:00",
            "y": 3
          }, {"x": "21:00", "y": 18}, {"x": "22:00", "y": 12}, {"x": "23:00", "y": 15}]
        }, {
          "name": "Wednesday",
          "data": [{"x": "0:00", "y": 46}, {"x": "1:00", "y": 15}, {"x": "2:00", "y": 25}, {
            "x": "3:00",
            "y": 19
          }, {"x": "4:00", "y": 14}, {"x": "5:00", "y": 26}, {"x": "6:00", "y": 27}, {
            "x": "7:00",
            "y": 11
          }, {"x": "8:00", "y": 22}, {"x": "9:00", "y": 34}, {"x": "10:00", "y": 0}, {
            "x": "11:00",
            "y": 10
          }, {"x": "12:00", "y": 6}, {"x": "13:00", "y": 25}, {"x": "14:00", "y": 4}, {
            "x": "15:00",
            "y": 1
          }, {"x": "16:00", "y": 10}, {"x": "17:00", "y": 36}, {"x": "18:00", "y": 28}, {
            "x": "19:00",
            "y": 1
          }, {"x": "20:00", "y": 9}, {"x": "21:00", "y": 2}, {"x": "22:00", "y": 4}, {"x": "23:00", "y": 41}]
        }, {
          "name": "Thursday",
          "data": [{"x": "0:00", "y": 40}, {"x": "1:00", "y": 21}, {"x": "2:00", "y": 8}, {
            "x": "3:00",
            "y": 16
          }, {"x": "4:00", "y": 28}, {"x": "5:00", "y": 23}, {"x": "6:00", "y": 8}, {
            "x": "7:00",
            "y": 26
          }, {"x": "8:00", "y": 19}, {"x": "9:00", "y": 6}, {"x": "10:00", "y": 42}, {
            "x": "11:00",
            "y": 36
          }, {"x": "12:00", "y": 3}, {"x": "13:00", "y": 26}, {"x": "14:00", "y": 2}, {
            "x": "15:00",
            "y": 27
          }, {"x": "16:00", "y": 23}, {"x": "17:00", "y": 18}, {"x": "18:00", "y": 23}, {
            "x": "19:00",
            "y": 25
          }, {"x": "20:00", "y": 22}, {"x": "21:00", "y": 50}, {"x": "22:00", "y": 25}, {"x": "23:00", "y": 5}]
        }, {
          "name": "Friday",
          "data": [{"x": "0:00", "y": 16}, {"x": "1:00", "y": 16}, {"x": "2:00", "y": 22}, {
            "x": "3:00",
            "y": 20
          }, {"x": "4:00", "y": 28}, {"x": "5:00", "y": 10}, {"x": "6:00", "y": 4}, {
            "x": "7:00",
            "y": 15
          }, {"x": "8:00", "y": 4}, {"x": "9:00", "y": 4}, {"x": "10:00", "y": 30}, {
            "x": "11:00",
            "y": 1
          }, {"x": "12:00", "y": 48}, {"x": "13:00", "y": 6}, {"x": "14:00", "y": 22}, {
            "x": "15:00",
            "y": 10
          }, {"x": "16:00", "y": 23}, {"x": "17:00", "y": 16}, {"x": "18:00", "y": 10}, {
            "x": "19:00",
            "y": 4
          }, {"x": "20:00", "y": 10}, {"x": "21:00", "y": 24}, {"x": "22:00", "y": 22}, {"x": "23:00", "y": 7}]
        }, {
          "name": "Saturday",
          "data": [{"x": "0:00", "y": 21}, {"x": "1:00", "y": 22}, {"x": "2:00", "y": 0}, {
            "x": "3:00",
            "y": 27
          }, {"x": "4:00", "y": 28}, {"x": "5:00", "y": 8}, {"x": "6:00", "y": 20}, {
            "x": "7:00",
            "y": 10
          }, {"x": "8:00", "y": 0}, {"x": "9:00", "y": 2}, {"x": "10:00", "y": 30}, {
            "x": "11:00",
            "y": 12
          }, {"x": "12:00", "y": 30}, {"x": "13:00", "y": 8}, {"x": "14:00", "y": 17}, {
            "x": "15:00",
            "y": 6
          }, {"x": "16:00", "y": 25}, {"x": "17:00", "y": 26}, {"x": "18:00", "y": 17}, {
            "x": "19:00",
            "y": 11
          }, {"x": "20:00", "y": 30}, {"x": "21:00", "y": 32}, {"x": "22:00", "y": 8}, {"x": "23:00", "y": 15}]
        }, {
          "name": "Sunday",
          "data": [{"x": "0:00", "y": 46}, {"x": "1:00", "y": 9}, {"x": "2:00", "y": 16}, {
            "x": "3:00",
            "y": 3
          }, {"x": "4:00", "y": 12}, {"x": "5:00", "y": 8}, {"x": "6:00", "y": 10}, {
            "x": "7:00",
            "y": 18
          }, {"x": "8:00", "y": 12}, {"x": "9:00", "y": 47}, {"x": "10:00", "y": 0}, {
            "x": "11:00",
            "y": 13
          }, {"x": "12:00", "y": 27}, {"x": "13:00", "y": 10}, {"x": "14:00", "y": 13}, {
            "x": "15:00",
            "y": 40
          }, {"x": "16:00", "y": 15}, {"x": "17:00", "y": 18}, {"x": "18:00", "y": 15}, {
            "x": "19:00",
            "y": 14
          }, {"x": "20:00", "y": 5}, {"x": "21:00", "y": 7}, {"x": "22:00", "y": 25}, {"x": "23:00", "y": 47}]
        }],
        isFetching: false
      }
    });
    component = renderer.create(
      <Provider store={store}>
        <DeviceHour/>
      </Provider>
    );
  })
  it('should render Device hour with given state from Redux store', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
})