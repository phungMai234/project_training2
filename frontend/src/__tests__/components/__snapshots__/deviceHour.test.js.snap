// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Device hour Component should render Device hour with given state from Redux store 1`] = `
<div
  className="device-hour"
>
  <label
    className="text-secondary"
  >
    Device by hour
  </label>
  <div
    className="row"
  >
    <div
      className="col-sm-9"
    >
      <div
        className="row justify-content-center p-0"
      >
        <div
          height="260"
          options={
            Object {
              "colors": Array [
                "#9c33ff",
              ],
              "dataLabels": Object {
                "enabled": false,
              },
              "legend": Object {
                "show": false,
              },
              "plotOptions": Object {
                "heatmap": Object {
                  "radius": 2,
                },
              },
              "responsive": Array [
                Object {
                  "breakpoint": 500,
                  "options": Object {
                    "plotOptions": Object {
                      "heatmap": Object {
                        "horizontal": false,
                      },
                    },
                  },
                },
              ],
              "tooltip": Object {
                "shared": false,
                "x": Object {
                  "show": true,
                },
              },
              "xaxis": Object {
                "axisBorder": Object {
                  "show": false,
                },
                "axisTicks": Object {
                  "show": false,
                },
                "convertedCatToNumeric": false,
                "labels": Object {
                  "show": true,
                },
              },
              "yaxis": Object {
                "axisBorder": Object {
                  "show": false,
                },
                "labels": Object {
                  "show": false,
                },
              },
            }
          }
          series={
            Array [
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 46,
                  },
                  Object {
                    "x": "1:00",
                    "y": 9,
                  },
                  Object {
                    "x": "2:00",
                    "y": 16,
                  },
                  Object {
                    "x": "3:00",
                    "y": 3,
                  },
                  Object {
                    "x": "4:00",
                    "y": 12,
                  },
                  Object {
                    "x": "5:00",
                    "y": 8,
                  },
                  Object {
                    "x": "6:00",
                    "y": 10,
                  },
                  Object {
                    "x": "7:00",
                    "y": 18,
                  },
                  Object {
                    "x": "8:00",
                    "y": 12,
                  },
                  Object {
                    "x": "9:00",
                    "y": 47,
                  },
                  Object {
                    "x": "10:00",
                    "y": 0,
                  },
                  Object {
                    "x": "11:00",
                    "y": 13,
                  },
                  Object {
                    "x": "12:00",
                    "y": 27,
                  },
                  Object {
                    "x": "13:00",
                    "y": 10,
                  },
                  Object {
                    "x": "14:00",
                    "y": 13,
                  },
                  Object {
                    "x": "15:00",
                    "y": 40,
                  },
                  Object {
                    "x": "16:00",
                    "y": 15,
                  },
                  Object {
                    "x": "17:00",
                    "y": 18,
                  },
                  Object {
                    "x": "18:00",
                    "y": 15,
                  },
                  Object {
                    "x": "19:00",
                    "y": 14,
                  },
                  Object {
                    "x": "20:00",
                    "y": 5,
                  },
                  Object {
                    "x": "21:00",
                    "y": 7,
                  },
                  Object {
                    "x": "22:00",
                    "y": 25,
                  },
                  Object {
                    "x": "23:00",
                    "y": 47,
                  },
                ],
                "name": "Sunday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 21,
                  },
                  Object {
                    "x": "1:00",
                    "y": 22,
                  },
                  Object {
                    "x": "2:00",
                    "y": 0,
                  },
                  Object {
                    "x": "3:00",
                    "y": 27,
                  },
                  Object {
                    "x": "4:00",
                    "y": 28,
                  },
                  Object {
                    "x": "5:00",
                    "y": 8,
                  },
                  Object {
                    "x": "6:00",
                    "y": 20,
                  },
                  Object {
                    "x": "7:00",
                    "y": 10,
                  },
                  Object {
                    "x": "8:00",
                    "y": 0,
                  },
                  Object {
                    "x": "9:00",
                    "y": 2,
                  },
                  Object {
                    "x": "10:00",
                    "y": 30,
                  },
                  Object {
                    "x": "11:00",
                    "y": 12,
                  },
                  Object {
                    "x": "12:00",
                    "y": 30,
                  },
                  Object {
                    "x": "13:00",
                    "y": 8,
                  },
                  Object {
                    "x": "14:00",
                    "y": 17,
                  },
                  Object {
                    "x": "15:00",
                    "y": 6,
                  },
                  Object {
                    "x": "16:00",
                    "y": 25,
                  },
                  Object {
                    "x": "17:00",
                    "y": 26,
                  },
                  Object {
                    "x": "18:00",
                    "y": 17,
                  },
                  Object {
                    "x": "19:00",
                    "y": 11,
                  },
                  Object {
                    "x": "20:00",
                    "y": 30,
                  },
                  Object {
                    "x": "21:00",
                    "y": 32,
                  },
                  Object {
                    "x": "22:00",
                    "y": 8,
                  },
                  Object {
                    "x": "23:00",
                    "y": 15,
                  },
                ],
                "name": "Saturday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 16,
                  },
                  Object {
                    "x": "1:00",
                    "y": 16,
                  },
                  Object {
                    "x": "2:00",
                    "y": 22,
                  },
                  Object {
                    "x": "3:00",
                    "y": 20,
                  },
                  Object {
                    "x": "4:00",
                    "y": 28,
                  },
                  Object {
                    "x": "5:00",
                    "y": 10,
                  },
                  Object {
                    "x": "6:00",
                    "y": 4,
                  },
                  Object {
                    "x": "7:00",
                    "y": 15,
                  },
                  Object {
                    "x": "8:00",
                    "y": 4,
                  },
                  Object {
                    "x": "9:00",
                    "y": 4,
                  },
                  Object {
                    "x": "10:00",
                    "y": 30,
                  },
                  Object {
                    "x": "11:00",
                    "y": 1,
                  },
                  Object {
                    "x": "12:00",
                    "y": 48,
                  },
                  Object {
                    "x": "13:00",
                    "y": 6,
                  },
                  Object {
                    "x": "14:00",
                    "y": 22,
                  },
                  Object {
                    "x": "15:00",
                    "y": 10,
                  },
                  Object {
                    "x": "16:00",
                    "y": 23,
                  },
                  Object {
                    "x": "17:00",
                    "y": 16,
                  },
                  Object {
                    "x": "18:00",
                    "y": 10,
                  },
                  Object {
                    "x": "19:00",
                    "y": 4,
                  },
                  Object {
                    "x": "20:00",
                    "y": 10,
                  },
                  Object {
                    "x": "21:00",
                    "y": 24,
                  },
                  Object {
                    "x": "22:00",
                    "y": 22,
                  },
                  Object {
                    "x": "23:00",
                    "y": 7,
                  },
                ],
                "name": "Friday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 40,
                  },
                  Object {
                    "x": "1:00",
                    "y": 21,
                  },
                  Object {
                    "x": "2:00",
                    "y": 8,
                  },
                  Object {
                    "x": "3:00",
                    "y": 16,
                  },
                  Object {
                    "x": "4:00",
                    "y": 28,
                  },
                  Object {
                    "x": "5:00",
                    "y": 23,
                  },
                  Object {
                    "x": "6:00",
                    "y": 8,
                  },
                  Object {
                    "x": "7:00",
                    "y": 26,
                  },
                  Object {
                    "x": "8:00",
                    "y": 19,
                  },
                  Object {
                    "x": "9:00",
                    "y": 6,
                  },
                  Object {
                    "x": "10:00",
                    "y": 42,
                  },
                  Object {
                    "x": "11:00",
                    "y": 36,
                  },
                  Object {
                    "x": "12:00",
                    "y": 3,
                  },
                  Object {
                    "x": "13:00",
                    "y": 26,
                  },
                  Object {
                    "x": "14:00",
                    "y": 2,
                  },
                  Object {
                    "x": "15:00",
                    "y": 27,
                  },
                  Object {
                    "x": "16:00",
                    "y": 23,
                  },
                  Object {
                    "x": "17:00",
                    "y": 18,
                  },
                  Object {
                    "x": "18:00",
                    "y": 23,
                  },
                  Object {
                    "x": "19:00",
                    "y": 25,
                  },
                  Object {
                    "x": "20:00",
                    "y": 22,
                  },
                  Object {
                    "x": "21:00",
                    "y": 50,
                  },
                  Object {
                    "x": "22:00",
                    "y": 25,
                  },
                  Object {
                    "x": "23:00",
                    "y": 5,
                  },
                ],
                "name": "Thursday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 46,
                  },
                  Object {
                    "x": "1:00",
                    "y": 15,
                  },
                  Object {
                    "x": "2:00",
                    "y": 25,
                  },
                  Object {
                    "x": "3:00",
                    "y": 19,
                  },
                  Object {
                    "x": "4:00",
                    "y": 14,
                  },
                  Object {
                    "x": "5:00",
                    "y": 26,
                  },
                  Object {
                    "x": "6:00",
                    "y": 27,
                  },
                  Object {
                    "x": "7:00",
                    "y": 11,
                  },
                  Object {
                    "x": "8:00",
                    "y": 22,
                  },
                  Object {
                    "x": "9:00",
                    "y": 34,
                  },
                  Object {
                    "x": "10:00",
                    "y": 0,
                  },
                  Object {
                    "x": "11:00",
                    "y": 10,
                  },
                  Object {
                    "x": "12:00",
                    "y": 6,
                  },
                  Object {
                    "x": "13:00",
                    "y": 25,
                  },
                  Object {
                    "x": "14:00",
                    "y": 4,
                  },
                  Object {
                    "x": "15:00",
                    "y": 1,
                  },
                  Object {
                    "x": "16:00",
                    "y": 10,
                  },
                  Object {
                    "x": "17:00",
                    "y": 36,
                  },
                  Object {
                    "x": "18:00",
                    "y": 28,
                  },
                  Object {
                    "x": "19:00",
                    "y": 1,
                  },
                  Object {
                    "x": "20:00",
                    "y": 9,
                  },
                  Object {
                    "x": "21:00",
                    "y": 2,
                  },
                  Object {
                    "x": "22:00",
                    "y": 4,
                  },
                  Object {
                    "x": "23:00",
                    "y": 41,
                  },
                ],
                "name": "Wednesday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 23,
                  },
                  Object {
                    "x": "1:00",
                    "y": 12,
                  },
                  Object {
                    "x": "2:00",
                    "y": 26,
                  },
                  Object {
                    "x": "3:00",
                    "y": 20,
                  },
                  Object {
                    "x": "4:00",
                    "y": 35,
                  },
                  Object {
                    "x": "5:00",
                    "y": 5,
                  },
                  Object {
                    "x": "6:00",
                    "y": 6,
                  },
                  Object {
                    "x": "7:00",
                    "y": 15,
                  },
                  Object {
                    "x": "8:00",
                    "y": 17,
                  },
                  Object {
                    "x": "9:00",
                    "y": 14,
                  },
                  Object {
                    "x": "10:00",
                    "y": 17,
                  },
                  Object {
                    "x": "11:00",
                    "y": 3,
                  },
                  Object {
                    "x": "12:00",
                    "y": 28,
                  },
                  Object {
                    "x": "13:00",
                    "y": 44,
                  },
                  Object {
                    "x": "14:00",
                    "y": 29,
                  },
                  Object {
                    "x": "15:00",
                    "y": 6,
                  },
                  Object {
                    "x": "16:00",
                    "y": 34,
                  },
                  Object {
                    "x": "17:00",
                    "y": 17,
                  },
                  Object {
                    "x": "18:00",
                    "y": 9,
                  },
                  Object {
                    "x": "19:00",
                    "y": 14,
                  },
                  Object {
                    "x": "20:00",
                    "y": 3,
                  },
                  Object {
                    "x": "21:00",
                    "y": 18,
                  },
                  Object {
                    "x": "22:00",
                    "y": 12,
                  },
                  Object {
                    "x": "23:00",
                    "y": 15,
                  },
                ],
                "name": "Tuesday",
              },
              Object {
                "data": Array [
                  Object {
                    "x": "0:00",
                    "y": 4,
                  },
                  Object {
                    "x": "1:00",
                    "y": 34,
                  },
                  Object {
                    "x": "2:00",
                    "y": 1,
                  },
                  Object {
                    "x": "3:00",
                    "y": 0,
                  },
                  Object {
                    "x": "4:00",
                    "y": 13,
                  },
                  Object {
                    "x": "5:00",
                    "y": 15,
                  },
                  Object {
                    "x": "6:00",
                    "y": 13,
                  },
                  Object {
                    "x": "7:00",
                    "y": 13,
                  },
                  Object {
                    "x": "8:00",
                    "y": 14,
                  },
                  Object {
                    "x": "9:00",
                    "y": 21,
                  },
                  Object {
                    "x": "10:00",
                    "y": 29,
                  },
                  Object {
                    "x": "11:00",
                    "y": 4,
                  },
                  Object {
                    "x": "12:00",
                    "y": 16,
                  },
                  Object {
                    "x": "13:00",
                    "y": 5,
                  },
                  Object {
                    "x": "14:00",
                    "y": 11,
                  },
                  Object {
                    "x": "15:00",
                    "y": 9,
                  },
                  Object {
                    "x": "16:00",
                    "y": 20,
                  },
                  Object {
                    "x": "17:00",
                    "y": 16,
                  },
                  Object {
                    "x": "18:00",
                    "y": 17,
                  },
                  Object {
                    "x": "19:00",
                    "y": 42,
                  },
                  Object {
                    "x": "20:00",
                    "y": 1,
                  },
                  Object {
                    "x": "21:00",
                    "y": 14,
                  },
                  Object {
                    "x": "22:00",
                    "y": 10,
                  },
                  Object {
                    "x": "23:00",
                    "y": 4,
                  },
                ],
                "name": "Monday",
              },
            ]
          }
          type="heatmap"
          width="500"
        />
        <div
          className="progress-heatmap"
        />
        <div
          className="number-progress"
        >
          <div>
            0
          </div>
          <div>
            10
          </div>
          <div>
            20
          </div>
          <div>
            30
          </div>
          <div>
            40
          </div>
          <div>
            50
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-sm-3"
    >
      <div
        height="260"
        options={
          Object {
            "grid": Object {
              "show": false,
            },
            "plotOptions": Object {
              "bar": Object {
                "background": "#f2f2f2",
                "barHeight": "90%%",
                "colors": Object {
                  "backgroundBarColors": Array [
                    "#f2f2f2",
                  ],
                  "backgroundBarOpacity": 1,
                  "backgroundBarRadius": 2,
                  "ranges": Array [
                    Object {
                      "color": "#00cc99",
                      "from": 0,
                      "to": 1000,
                    },
                  ],
                },
                "endingShape": "rounded",
                "horizontal": true,
                "startingShape": "flat",
              },
            },
            "xaxis": Object {
              "axisBorder": Object {
                "show": false,
              },
              "axisTicks": Object {
                "show": false,
                "width": 522,
              },
              "convertedCatToNumeric": false,
              "labels": Object {
                "show": false,
              },
            },
            "yaxis": Object {
              "axisTicks": Object {
                "show": false,
              },
              "labels": Object {
                "style": Object {
                  "colors": Array [],
                  "fontFamily": "Times New Roman",
                  "fontSize": "14px",
                },
              },
              "max": 522,
              "min": 0,
            },
          }
        }
        series={
          Array [
            Object {
              "data": Array [
                Object {
                  "x": "Mon",
                  "y": 430,
                },
                Object {
                  "x": "Tue",
                  "y": 405,
                },
                Object {
                  "x": "Wed",
                  "y": 372,
                },
                Object {
                  "x": "Thu",
                  "y": 522,
                },
                Object {
                  "x": "Fri",
                  "y": 416,
                },
                Object {
                  "x": "Sat",
                  "y": 422,
                },
                Object {
                  "x": "Sun",
                  "y": 326,
                },
              ],
            },
          ]
        }
        type="bar"
        width="100%"
      />
    </div>
  </div>
</div>
`;
